tests/fixtures/controllers/securityController.ts

```diff
@@ -4,6 +4,24 @@ import * as koa from 'koa';
 import { Get, Request, Response, Route, Security } from '../../../src';
 import { ErrorResponseModel, UserResponseModel } from '../../fixtures/testModel';
 
+declare module 'express' {
```

Comment:
If this is the only file that uses it, then do we even need to `declare module` at all?

Suggested change:
```ts
interface RequestWithUser {
  user?: any;
}

//...

public async GetWithApiForHapi(@Request() request: RequestWithUser): Promise<UserResponseModel> {
    return Promise.resolve(request.user);
}
```

Reply:
Good idea. I already had a hunch that this could be done cleaner but this didn't occur to me (in my defense I was probably very tired ;)).

I have now implemented this change.